app/src/forms/admin/routes.js

```diff
@@ -18,22 +18,172 @@ routes.use(currentUser);
 //
 // Forms
 //
+
+/**
+ * @openapi
+ * /admin/forms:
+ *  get:
+ *    tags:
+ *      - Admin
+ *    description: This endpoint will fetch all forms under the admin access.
+ *    security:
+ *      - bearerAuth: []
+ *      - basicAuth: []
+ *      - openId: []
```

Comment:
Are these permission correct? The old docs say that it based on the JWT scope "admin".